server/controllers/events/events_controller.go

```diff
@@ -244,7 +245,10 @@ func (e *VCSEventsController) handleBitbucketCloudPost(w http.ResponseWriter, r
 func (e *VCSEventsController) handleBitbucketServerPost(w http.ResponseWriter, r *http.Request) {
 	eventType := r.Header.Get(bitbucketEventTypeHeader)
 	reqID := r.Header.Get(bitbucketServerRequestIDHeader)
-	sig := r.Header.Get(bitbucketServerSignatureHeader)
+	sig := r.Header.Get(bitbucketServerSignatureHeader256)
+	if sig == "" {
+		sig = r.Header.Get(bitbucketServerSignatureHeader)
+	}
```

Comment:
why is this only added to only bitbucket?

Reply:
your description talks about github and your tests are github specific, so I'm bit confused